cluster/manifests/external-dns/deployment.yaml

```diff
@@ -26,12 +26,12 @@ spec:
          operator: Exists
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.4
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.5
         args:
         - --source=service
         - --source=ingress
         - --provider=aws
         - --registry=txt
         - --txt-owner-id={{ .Region }}:{{ .LocalID }}
         - --compatibility=mate # remove when we switched to the new annotations
-        - --debug # remove when we are sure external-dns can be trusted
+        - --log-level=debug # remove when we are sure external-dns can be trusted
```

Comment:
@linki Do we really do not trust it or is it just a leftover?
I mean we run it in production so we trust it.

Reply:
I didn't want to make this decision at the time. Since this update is more dangerous than the ones before I would leave this flag on debug in this PR.